src/util.cpp

```diff
@@ -1131,6 +1131,27 @@ std::string toString(const Decl *decl)
     else if (dyn_cast<TypeDecl>(decl))
         return "TypeDecl : NamedDecl";
     // ...
+#if LLVM_VERSION_AT_LEAST(6,0,0)
+    else if (dyn_cast<DecompositionDecl>(decl))
+        return "DecompositionDecl : VarDecl : DeclaratorDecl : ValueDecl : "
+               "NamedDecl";
+#endif
+    else if (dyn_cast<ImplicitParamDecl>(decl))
+        return "ImplicitParamDecl : VarDecl : DeclaratorDecl : ValueDecl : "
+               "NamedDecl";
+    else if (dyn_cast<ParmVarDecl>(decl))
+        return "ParmVarDecl : VarDecl : DeclaratorDecl : ValueDecl : NamedDecl";
+    else if (dyn_cast<VarTemplateSpecializationDecl>(decl))
+        return "VarTemplateSpecializationDecl : VarDecl : DeclaratorDecl : "
+               "ValueDecl : NamedDecl";
+    else if (dyn_cast<VarDecl>(decl))
+        return "VarDecl : DeclaratorDecl : ValueDecl : NamedDecl";
+    else if (dyn_cast<DeclaratorDecl>(decl))
+        return "DeclaratorDecl : ValueDecl : NamedDecl";
+    // ...
```

Comment:
What does this ... mean?

Reply:
That denotes that there are more types between previous and next types according to the LLVM [manual](https://clang.llvm.org/doxygen/classclang_1_1Decl.html). This is because I'm lazy to add all the types. :sweat:

Reply:
Ok, I think just put a small comment saying there are more types that are not handled.